Sources/Jobs/Scheduler.swift

```diff
@@ -103,7 +106,7 @@ private extension Scheduler {
                                                        _ keyPath: KeyPath<K, T>) throws -> JobConfiguration {
         let identifier = ObjectIdentifier(keyPath)
         let jobConfiguration = try JobConfiguration(SwifCron(cronTrigger))
-        EnvironmentValues.shared.values[identifier] = job
+        _ = EnvironmentValue(keyPath, job)
```

Comment:
would this work?
```suggestion
        EnvironmentValue(keyPath, job)
```

Reply:
I made EnvironmentValue `@discardableResult`